### ISSUE TITLE
How is this being used?

### ISSUE THREAD

**Issue by pmackay** [openvocab/ovn#18]
Would it be possible to add some info in the README describing how this vocab is or is intended to be used and where? Just for some useful context :)

**Comment by bhaugen** [comment]
1. Being used in total: see https://github.com/valnet/valuenetwork

Being used partially: https://github.com/openvocab/holodex
App in progress:
https://github.com/open-app/holodex
1. Intended to be used: see https://github.com/openvocab/ovn/issues/12

(Which will be turned into a simpler list, without all the surrounding chatter, soon.)

Will add to README by tomorrow.

**Comment by bhaugen** [comment]
Resolved, I think. @pmackay - what do you think?

**Comment by pmackay** [comment]
Yes. There are more things I'm wondering about but this helps :)

**Comment by bhaugen** [comment]
I have a bit more to add to the readme, but haven't finished it yet. You can read it in my fork:
https://github.com/valnet/ovn

If you have other specific or even vague questions, please re-open this issue and add them, or start another issue.

I would also be curious about your interest: why are you interested in this vocab (assuming you are)? Can you see connections to other things you are doing? Might you be interested in using this vocab (when it becomes usable) or participating in this effort to help it become usable?

**Comment by pmackay** [comment]
Couple of things:

Am interested in the physical flow of resources in a community, particularly w.r.t. the circular economy. Am participating in https://oscedays.org/ and discovered http://ipotables.net recently where a discussion there brought me back to here (which I've seen before from your discussions with Loomio, etc). 

Also doing this project https://plus.google.com/u/0/communities/109044086640584910571, working with Open Referral that defines a model with overlaps in places with this one.

**Comment by bhaugen** [comment]
Welcome! Lotta connections here. I love "Connecting communities for greater resilience".

You might be interested in this: http://locecon.org/

It takes this same model up to another level, from the operations of economic networks (the OVN level) to regional aggregated resource flows. One of our goals is to connect both levels.

We would also like to integrate with something like Odum's ecosystem modeling Energese vocab:
http://en.wikipedia.org/wiki/Howard_T._Odum#Ecological_modeling

Does any of that make sense to you? Or, where would you like to take this conversation next?
